src/System.Memory/src/System.Memory.csproj

```diff
@@ -27,12 +24,22 @@
     <Compile Include="System\SpanHelpers.Clear.cs" />
     <Compile Include="System\ThrowHelper.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
The netstandard configuration is needed because there are two distinct assemblies being generated: the "fast span assembly" with references to System.Private.CoreLib's private implementation, and the "slow span assembly" with no runtime dependencies.